tldw_Server_API/app/core/AuthNZ/create_admin.py

```
@@ -0,0 +1,160 @@
+                expires_in_days=365,
+            )
+            print(f"[create-admin] Admin API key created (expires in 365 days).")
+            # Intentionally do NOT print the key to logs in Docker context for security
+        except Exception as api_err:
+            # Non-fatal: user can generate API keys later via the UI/API
+            logger.debug(f"[create-admin] API key creation skipped: {api_err}")
+            print("[create-admin] Warning: could not auto-generate API key (user can create one later).")
+
+        return True
+
+    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)